README.md

```diff
@@ -591,6 +591,14 @@ Or run any command using the application environment:
 
     honcho run ./manage.py <command>
 
+### Available commands
+
+**activity_csv**: Collect and produce a CSV containing usage information from
```

Comment:
@bdero Nit: Add backticks so the command name displays like this: **`activity_csv`**